Angular/Angular guidelines and best practices/Testing.md

```diff
@@ -531,6 +531,96 @@ beforeEach(async(() => {
 }));
 ```
 
+This is, arguably, not the best solution but it might be the simplest solution and good enough for some cases.
+
+### Alternative approach with a host/wrapper component
+
+Changing change detection strategy from OnPush to Default in tests is not ideal because that could theoretically make the component behave differently when comparing application runtime where it is using OnPush and tests where it is using Default CD.
```

Comment:
```suggestion
Changing the change detection strategy from OnPush to Default in tests is not ideal because that could theoretically make the component behave differently when comparing the application runtime, where it is using OnPush, and tests, where it is using the Default CD.
```